src/openshift/cluster.ts

```diff
@@ -641,9 +776,14 @@ export class Cluster extends OpenShiftItem {
      * - username string, or
      * - `null` in case of user cancelled (pressed `ESC`), or
      * - `undefined` if user pressed `Back` button
+     *
+     * @param clusterURL a cluster to login to
+     * @param assUserLaber a label to be shown for add new user action
```

Comment:
addUserLabel

It seems like there is a plugin for eslint that helps with JSDoc, I'll create an issue to consider adding it, because it could help out

Reply:
Hm. Sorry. 
ACK